### ISSUE TITLE
Gem is not requirable!

### ISSUE THREAD

**Issue by DenniJensen** [commonthread/flickr_fu#31]
I tried to require the gem by `requrire 'flickr_fu'` and this error came up.
![screen shot 2015-07-04 at 15 57 40](https://cloud.githubusercontent.com/assets/3010752/8508160/a0c5b1d8-2265-11e5-8190-5130ec6291c5.png)

**Comment by NishantUpadhyay-BTC** [comment]
Yes, I am facing the same issue +1. Do you find any solution for this?
